src/app/layout.tsx

```diff
@@ -0,0 +1,68 @@
+import Script from "next/script";
+import { Footer } from "../components/footer";
+import { Header } from "../components/header";
+import "./globals.css";
+import { Roboto_Flex, Roboto_Mono } from "next/font/google";
+import { Metadata } from "next";
+
+// Loads fonts: https://nextjs.org/docs/pages/building-your-application/optimizing/fonts
+export const robotoFlex = Roboto_Flex({ subsets: ["latin"], display: "swap" });
+export const robotoMono = Roboto_Mono({ subsets: ["latin"], display: "swap" });
+
+export default function Layout({ children }: { children: React.ReactNode }) {
+    return (
+        <html lang="en">
+            <Script id="google-tag-manager" strategy="afterInteractive">
+                {`(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+                new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+                j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+                '../www.googletagmanager.com/gtm5445.html?id='+i+dl;f.parentNode.insertBefore(j,f);
+                })(window,document,'script','dataLayer','GTM-WFBZNP9');`}
+            </Script>
+            <body className="bg-light-black">
+                <noscript>
+                    <iframe
+                        src="https://www.googletagmanager.com/ns.html?id=GTM-WFBZNP9"
+                        height="0"
+                        width="0"
+                        style={{ display: "none", visibility: "hidden" }}
+                    ></iframe>
+                </noscript>
+                <Header />
+                <div className="max-w-5xl mx-auto min-h-screen">{children}</div>
+                <Footer />
+            </body>
+        </html>
+    );
+}
+
+export const metadata: Metadata = {
```

Comment:
Did you double-check to get all metadata tags from current site?
Also, can we're overriding this per-checklist, right?